APCAppCore/APCAppCore/DataSubstrate/Model/APCUser.m

```diff
@@ -667,5 +669,28 @@ - (void)setSavedSharingScope:(NSNumber *)savedSharingScope
         [[NSUserDefaults standardUserDefaults] removeObjectForKey:kSavedSharingScopeKey];
     }
 }
+- (NSArray *)dataGroups {
+    return [[NSUserDefaults standardUserDefaults] objectForKey:kSavedDataGroupsKey];
+}
+
+- (void)setDataGroups:(NSArray *)dataGroups {
+    [[NSUserDefaults standardUserDefaults] setObject:dataGroups forKey:kSavedDataGroupsKey];
+    [[NSUserDefaults standardUserDefaults] synchronize];
+}
+
+- (NSString *)subpopulationGuid {
+    NSString *guid = [[NSUserDefaults standardUserDefaults] stringForKey:kSavedSubpopulationGuidKey];
```

Comment:
Would the subpopulation be considered sensitive info? (same question applies to dataGroups) if so, probably should store somewhere more secure than NSUserDefaults...